### PR TITLE
Updates FAQs

### LINKED ISSUE #5: Outdated FAQs
The FAQs are outdated and many links are dead.

### PR DESCRIPTION
**Pull request by AlexITA1100**
- Updated dead links;
- Updated server on Linux FAQs;
- Both FAQs have the same structure now;
- Added table for code licenses;
- Added Italian localisation for the FAQs;

Resolves #5

### CODE REVIEW

**Review by Starystars67 — APPROVED** [review]
Thank you @AlexITA1100 for helping with a lot of this kind of work at the moment. 
<3

File: docs/en/support/game-faq.md [hunk 1]
```diff
@@ -40,21 +38,20 @@ We are working hard to make performance as good as possible. Sadly, there isn't
 #### **Where can I find the code?**
 
 All the source code can be found on our [GitHub](https://github.com/BeamMP).
+Before doing anything keep in mind that the code is under [Terms of Use](https://forum.beammp.com/t/terms-of-use-v1-0/43).
```
Comment (20dka) [inline docs/en/support/game-faq.md:41]: I'm not sure about the wording here. Could you maybe change it to:

> Before doing anything keep in mind that the code is subject to our Terms of Use and Licenses:

Thanks.

File: docs/en/support/server-faq.md [hunk 2]
```diff
@@ -7,13 +7,34 @@ List of commonly asked questions and known bugs.
...
+We provide binaries for many Linux distributions [here](https://github.com/BeamMP/BeamMP-Server/releases/latest). If there are no binaries for your distribution, you can build it yourself you can do it by downloading the source on our [GitHub](https://github.com/BeamMP/BeamMP-Server), a tutorial can be found [here](https://github.com/BeamMP/BeamMP-Server#build-instructions).
 
 ---
+## **Miscellaneous**
 
+---
+#### **Where can I find the code?**
+
+All the source code can be found on our [GitHub](https://github.com/BeamMP).
+Before doing anything keep in mind that the code is under [Terms of Use](https://forum.beammp.com/t/terms-of-use-v1-0/43).
```
Comment (20dka) [inline docs/en/support/server-faq.md:24]: see other comment

**Review by 20dka — CHANGES_REQUESTED** [review]

**Review by Starystars67 — APPROVED** [review]
This looks much better too!

File: docs/en/support/server-faq.md [hunk 3]
```diff
@@ -7,13 +7,33 @@ List of commonly asked questions and known bugs.
 ---
 #### **How can I setup my own servers**
 
-All the information to setup your own server can be found [here](../create-a-server).
+All the information to setup your own server can be found [here](https://docs.beammp.com/server/create-a-server/).
```
Comment (lionkor) [inline docs/en/support/server-faq.md:10]:
```suggestion
All the information to set up your own server can be found [here](https://docs.beammp.com/server/create-a-server/).
```

**Review by lionkor — CHANGES_REQUESTED** [review]

**Review by Starystars67 — APPROVED** [review]

**Review by Starystars67 — APPROVED** [review]

**Review by lionkor — APPROVED** [review]

**Pull request merged**